### PR TITLE
i18n(es): Updated `astro-components.mdx` & `docker.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `astro-components.mdx` & `docker.mdx` based on this 2 commits 063da61a0f73ecf5bc6030ff1b4d2484633b1faa & 365e291af204a5840aa9b16861251ee85ac52509

### CODE REVIEW

**Review by ElianCodes — COMMENTED** [review]
LGTM on the `docker` part!

File: src/content/docs/es/core-concepts/astro-components.mdx [hunk 1]
```diff
@@ -277,6 +277,55 @@ const { titulo } = Astro.props
...
+import BaseLayout from "./BaseLayout.astro";
+---
+<BaseLayout>
+  <slot name="head" slot="head"/>
+  <slot />
+</BaseLayout>
+```
+
+:::note
+Slots con nombre pueden ser transferidos a otro componente usando los atributos `name` y `slot` en una etiqueta `<slot />`
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-components.mdx:314]:
```suggestion
Los slots con nombre pueden ser transferidos a otro componente usando los atributos `name` y `slot` en una etiqueta `<slot />`
```

File: src/content/docs/es/core-concepts/astro-components.mdx [hunk 2]
```diff
@@ -277,6 +277,55 @@ const { titulo } = Astro.props
 </div>
 ```
 
+### Transferir slots
+
+Los slots pueden ser transferidos a otros componentes. Por ejemplo, cuando se crean layouts anidados:
```
Comment (Waxer59) [inline src/content/docs/es/core-concepts/astro-components.mdx:282]:
```suggestion
Los slots pueden ser transferidos a otros componentes. Por ejemplo, cuando se crean diseños anidados:
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

File: src/content/docs/es/core-concepts/astro-components.mdx (hunk 2 above)
Comment (dreyfus92) [reply src/content/docs/es/core-concepts/astro-components.mdx:282]: aquí lo voy a cambiar a `plantillas`

Comment (Waxer59) [reply src/content/docs/es/core-concepts/astro-components.mdx:282]: Ok, sin problema

Comment (Waxer59) [reply src/content/docs/es/core-concepts/astro-components.mdx:282]: queda perfecto también

**Comment by dreyfus92** [comment]
Muchas gracias por el review @Waxer59 😄

**Comment by dreyfus92** [comment]
Thank you as for the review @ElianCodes, welcome back 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Comment by ElianCodes** [comment]
As this was reviewed by myself and @Waxer59, I don't think we need explicit approval by Yan, so let's get this shipped! 🚀🛳️

**Pull request merged**